### DISCUSSION TITLE
Arc style favourites

### DISCUSSION THREAD

**Discussion by keifererikson** [zen-browser/desktop#916] [Ideas]
Just throwing this out there to see what everyone thinks. 

I would love to have favourites implemented in Zen, similar to Arc browser. Pinned tabs always have their place, but favourites really add another layer of usability. Having the favourite tab re-open to the same page on first open is really useful for my workflow, but I wonder if others would appreciate this in Zen, like I would.

This would also be complemented by the suggested feature discussed in #875, and the issue #238, where the favourites could be unique to each workspace.

**Comment by MillerApps** [comment]
I second this, as a user moving from Arc, the tabs and bookmarks of other browsers now feel alien to me. I would really love if the pinned tabs were more like Arc's as well. In Arc, it seems they are more or less treated as regular bookmarks. And really, the only other Arc feature I find myself reaching for is the quick URL copy ⇧ ⌘ c; that little feature is handy.

**Reply by KiKaraage** [reply]
For copying URL, [this addon](https://addons.mozilla.org/en-US/firefox/addon/copy-frame-or-page-url/) worked perfectly for me! It can be pinned to toolbar and support keyboard shortcut as well

**Comment by vicvancooten** [comment]
Agree. There is many reasons to prefer Zen over Arc, but Arc broke my brain by combining tabs & bookmarks and grouping those using workspaces. I've never used bookmarks or bookmark toolbars because they're conceptually a mess. Mixing bookmarks and tabs together and allowing them to be organized into folders is an actual game changer and I love it.

Since this is a new concept I have no clue how hard it would be to implement in Firefox, but technically they're just unloaded tabs I think. 

Obviously this feature would combine incredibly well with auto disappearing tabs under the line, but this isn't even required.

**Comment by alessandroberna** [comment]
In the [zen theme store](https://zen-browser.app/themes) there seem to be some themes that make the tab bar more "Arc like".
I haven't tried Arc myself, so I can't say if the browser behavior matches, but based on the screenshots I've found, [this theme](https://zen-browser.app/themes/ad97bb70-0066-4e42-9b5f-173a5e42c6fc) makes the tab bar look quite similar, at least visually
![image](https://github.com/user-attachments/assets/32e634b6-c44f-4d90-840b-3b3c98532bc8)

**Reply by vicvancooten** [reply]
[This one](https://zen-browser.app/themes/bed8c922-616a-4165-8c86-6822ccf478ad) goes even further! But that may even further the point. You can totally get Zen to look like Arc, but that doesn't mean if offers the amazing tab management options arc does. If it did, I'd switch primary I think.

**Reply by MillerApps** [reply]
@alessandroberna that's merely an UI update. What @vicvancooten and many others, I would assume, are looking for is a similar pin and overall tab experience to Arc as it definitely feels like the better way to handle tabs and bookmarks after using it for some time.

**Comment by erutan** [comment]
This is essentially a dupe, but I came across it first and it has an actual list of functional features in it.

https://github.com/zen-browser/desktop/discussions/2829
